clients/rancher/client.go

```diff
@@ -52,6 +53,7 @@ type Client struct {
 	// Flags is the environment flags used by the client to test selectively against a rancher instance.
 	Flags      *environmentflag.EnvironmentFlags
 	restConfig *rest.Config
+	CreatorID  string
```

Comment:
Should we call this UserID? 

Especially considering it'd look like this: `client.UserID`

Don't have strong opinions on naming though

Reply:
I have it as CreatorID simply because thats the naming convention that is used in rancher for a variety of objects. Figured i'd stick with the theme, if your ok with that.

Reply:
Updated it!